pulpcore/app/models/content.py

```diff
@@ -44,7 +44,7 @@ def bulk_get_or_create(self, objs, batch_size=None):
             for i in range(len(objs)):
                 try:
                     objs[i].save()
-                except ValueError or IntegrityError:
+                except (ValueError, IntegrityError):
```

Comment:
I think we can remove the ValueError exception handling. If it gets raised now, it means something did go wrong.

Reply:
... took me a while to realize that the syntax for catching manifold exceptions was wrong...